src/cr/cube/matrix/assembler.py

```diff
@@ -233,47 +233,64 @@ def means(self):
         """
         return self._assemble_matrix(self._measures.means.blocks)
 
-    def pairwise_indices(self, alpha, only_larger):
+    @lazyproperty
+    def pairwise_indices(self):
         """2D optional ndarray of tuple of int column-idxs pairwise-t threshold.
 
         Raises `ValueError if the cube-result does not include `overlaps` cube-measures.
         """
```

Comment:
What does `"2D optional ndarray"` mean? I think you must mean `"Optional 2D ndarray"`, but that would imply the value can be either `2D ndarray` or `None`. This was true of the prior implementation but no longer appears to be.

Also, I'm not clear why it would (unconditionally) `"Raise ValueError if the cube-result does not include overlaps cube-measures`". Aren't overlaps only required when the columns-dimension is MR?

While you're in there, make sure the back-ticks match, a closing one is missing after `ValueError` and also I don't know why it would be "cube-measures", overlaps is only one cube-measure, isn't it?